### ISSUE TITLE
rename group members to editors

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/adhocracy4#724]
**URL:** 
**user:** 
**expected behaviour:** I like to see the name of a role and immediately understand what the role is.
**behaviour:** group members is a bit vague, and not telling me what they are allowed to do
**important screensize:**
**device & browser:** 
**Comment/Question:** It only affects mB (hopefully).

Screenshot?

**Comment by fuzzylogic2000** [comment]
Hmm, not sure editor fully works. For the projects "project editor" would be fine. But what about the group members in an organisation that are allowed to create projects? Still "project editor", even though they are not allowed to change every project, but only the ones of their group? And in the user's model they are called groups by default. Maybe we just give the orga and project groups a verbose name and an helptext instead of changing everything?
@CarolingerSeilchenspringer and @sabinammm what do you think?

**Comment by sabinammm** [comment]
Yes, this is something that I've stumbled across as well. You can see in the wiki entry on roles that it's confusing, and I hope that I defined them correcty. Indeed I think that changing everything is tricky b/c - and this is my initial impression - roles do change as well, so that their names cannot do all the telling permanently and comprehensively. Thus, I think that giving verbose names and helptexts are the best way to go, as these can be adjusted if need be. But I think these are really necessary to counter confusion.

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 @sabinammm I see that groups maybe are not the best wording. They are more like restricted initiators, working on behalf of an organisation. On mB they are all "Dienstleister" for administration. We will also discuss their rights for plans again, so "project editor" wouldn't do. Question: Where would you work with verbose names and helptexts? As far as I know the namens "initiators" and "group/ group members" are not mentioned in frontend/dashboard anywhere. Only moderators, no?

**Comment by fuzzylogic2000** [comment]
when starting from the project, the renaming becomes a bit crazy here: https://github.com/liqd/adhocracy4/blob/de27b93a72784d9f3bc77c4830248062f7634f9b/adhocracy4/projects/predicates.py#L7
Sooo, I think, we have to fix #725 first.
